File: ch02/python/2.4.iter_to_recursive.py
print('Code in a loop')

i = 0

while i < 5:
    print(i, 'Hello')
    i += 1

def hello(i = 0):

    print(i, 'Hello')
    if i + 1 < 5:
        hello(i + 1)

print('Code in recursive function')
hello()

File: ch02/nodejs/2.4.iterToRecursive.js
console.log('Code in a loop');

let i = 0;

while (i < 5) {
    console.log(i, 'Hello');
    i++
}

const hello = i => {
    if (i === undefined) i = 0;
    console.log(i, 'Hello');

    i + 1 < 5 && hello(i + 1)
}

console.log('Code in Recursive function');
hello()